Samples/IntelliTect.TestTools.Data.Sample/SampleDataProject/SampleDataProject.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
```

Comment:
Do we dare turn on nullable reference types?

Reply:
I turned off Nullable for this project. I think it slightly muddies the example. Just trying to have as few moving parts as possible.